### ISSUE TITLE
Impossible to start server without setting server phone number. 

### ISSUE THREAD

**Issue by Civitello** [lukeaschenbrenner/TxtNet-Browser#11]
It is impossible to start the server without first setting a server phone number as that ui element is unavailable.

**Comment by lukeaschenbrenner** [comment]
Yes, this is a restriction caused by how closely coupled the server and client app modules are. As it stands right now, there are 3 options I could pursue:
1. Do nothing. The server phone number only needs to be a valid number for client requests, so someone only interested in setting up a server can quite easily input a fake number.
2. Create a separate app with only server functionality. This would require maintaining 2 separate versions of the same project, due to how SMS listeners have been implemented.
3. Add a button on the initial server phone number screen to run the app in server-only mode. This would restrict the app from attempting to load any client views and only allow the server screen to be used.

As far as these three options go, it seems to me that the third might be the best suited for your use case. Let me know if that is the kind of solution that you would be looking for.

**Comment by jbro129** [comment]
The third option is definitely the best.

**Comment by lukeaschenbrenner** [comment]
Thanks for the input - this should be an easy fix. I'll get it out in the next version (timeframe TBD).
